src/app/core/services/advance-request.service.ts

```
@@ -552,12 +550,17 @@ export class AdvanceRequestService {
       areq_source: advanceRequestPlatform.source,
       areq_state: this.getAdvanceRequestState(advanceRequestPlatform.state),
       areq_updated_at: advanceRequestPlatform.updated_at,
-      ou_department: advanceRequestPlatform.employee.department && advanceRequestPlatform.employee.department.name,
-      ou_department_id: advanceRequestPlatform.employee.department && advanceRequestPlatform.employee.department.id,
+      ou_department:
+        advanceRequestPlatform.employee &&
+        advanceRequestPlatform.employee.department &&
+        advanceRequestPlatform.employee.department.name,
+      ou_department_id: advanceRequestPlatform.employee && advanceRequestPlatform.employee.department.id,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)